galasa-parent/dev.galasa.framework.api.common/src/testFixtures/java/dev/galasa/framework/api/common/mocks/MockIConfigurationPropertyStoreService.java

```diff
@@ -36,6 +36,8 @@ public MockIConfigurationPropertyStoreService(@NotNull String namespace) {
             this.properties.put(namespace+".test.property.testing.local.stream", "value3");
             this.properties.put(namespace+".test.property.testing.stream", "value4");
             this.properties.put(namespace+".test.stream", "value5");
+        }else if (this.namespaceInput == "empty"){
+            //add no properties
         }else{
```

Comment:
I prefer spaces between brackets and else's `} else {` not having them looks odd.